src/pages/contact.js

```diff
@@ -31,6 +31,38 @@ function ContactPage() {
               <span className="centerStyles">Interested in Learning More About Empathy Bytes?</span>
             </h1>
           </div>
+          <div className="form-container">
+            <div style={{fontSize: '20px', textAlign: 'center', margin: '20px'}}>Complete this form and one of our team leads will reach out to you shortly!</div>
```

Comment:
This font size for "complete this form..." and the "Interested in Learning More About Empathy Bytes?" is too small I think. Also, it's best that we stick with either stylesheets or inner styling not both so I would recommend moving the inner styling to a css file.